hoomd/md/EvaluatorPairExpandedMie.h

```diff
@@ -0,0 +1,218 @@
+            v["n"] = npow;
+            v["m"] = mpow;
+
+            Scalar sigma = fast::pow(rep / att, 1 / (npow - mpow));
+            Scalar epsilon = rep / fast::pow(sigma, npow) * (npow - mpow) / npow
+                             * fast::pow(npow / mpow, mpow / (mpow - npow));
+
+            v["epsilon"] = epsilon;
+            v["sigma"] = sigma;
+
+            v["delta"] = delta;
+            return v;
+            }
+#endif
+        } __attribute__((aligned(16)));
+
+    //! Constructs the pair potential evaluator
+    /*! \param _rsq Squared distance between the particles
+        \param _rcutsq Squared distance at which the potential goes to 0
+        \param _n First, larger exponent that captures hard-core repulsion
+        \param _m Second, smaller exponent that captures attraction
+        \param _params Per type pair parameters of this potential
+        \param _delta Horizontal shift in r
+    */
+    DEVICE EvaluatorPairExpandedMie(Scalar _rsq, Scalar _rcutsq, const param_type& _params)
+        : rsq(_rsq), rcutsq(_rcutsq), rep(_params.rep), att(_params.att), npow(_params.npow),
+          mpow(_params.mpow), delta(_params.delta)
+        {
+        }
+
+    //! ExpandedMie doesn't use diameter
+    DEVICE static bool needsDiameter()
+        {
+        return false;
+        }
+    //! Accept the optional diameter values
+    /*! \param di Diameter of particle i
+        \param dj Diameter of particle j
+    */
+    DEVICE void setDiameter(Scalar di, Scalar dj) { }
+
+    //! ExpandedMie doesn't use charge
+    DEVICE static bool needsCharge()
+        {
+        return false;
+        }
+    //! Accept the optional diameter values
+    /*! \param qi Charge of particle i
+        \param qj Charge of particle j
+    */
+    DEVICE void setCharge(Scalar qi, Scalar qj) { }
+
+    //! Evaluate the force and energy
+    /*! \param force_divr Output parameter to write the computed force divided by r.
+        \param pair_eng Output parameter to write the computed pair energy
+        \param energy_shift If true, the potential must be shifted so that V(r) is continuous at the
+       cutoff \note There is no need to check if rsq < rcutsq in this method. Cutoff tests are
+       performed in PotentialPair.
+
+        \return True if they are evaluated or false if they are not because we are beyond the cutoff
+    */
+    DEVICE bool evalForceAndEnergy(Scalar& force_divr, Scalar& pair_eng, bool energy_shift)
+        {
+        // precompute some quantities
+        Scalar r = fast::sqrt(rsq);
+        Scalar rinv = fast::rsqrt(rsq);
+
+        // compute the force divided by r in force_divr
+        if (rsq < rcutsq && rep != 0)
+            {
+            Scalar rmd = r - delta;
+            Scalar rmdinv = Scalar(1.0) / rmd;
+            Scalar rmd2inv = rmdinv * rmdinv;
+            Scalar rmdninv = fast::pow(rmd2inv, npow / Scalar(2.0));
+            Scalar rmdminv = fast::pow(rmd2inv, mpow / Scalar(2.0));
```

Comment:
```suggestion
            Scalar rmdninv = fast::pow(rmd2inv, npow / Scalar(2.0));
```
These lines are the same.